### ISSUE TITLE
bug(developer,windows): 'Unknown Publisher' message displayed on the screen during the installation

### ISSUE THREAD

**Issue by bharanidharanj** [keymanapp/keyman#9692]
### Describe the bug

I observed that a warning message 'Unknown Publisher'  is shown on the screen when installing the Keyman Developer 17.0.186 build.

### Reproduce the bug

1. Download Keyman 17.0.186-alpha build.
2. Double-click on the Keyman 17.0.186-alpha.exe file. 

Here, I noticed that a warning message appears on the screen with the 'Run anyway' and 'Don't run' buttons. 

Note: I was able to reproduce this issue on Windows 11 OS (VM).

I have attached the Screenshots for reference. 

..Warning message in Windows 10 OS

   <img width="300" src="https://github.com/keymanapp/keyman/assets/19683143/642a0707-f7e2-431b-aba8-d0ce480290e7">

..Warning message in Windows 11 OS

   <img width="300" src="https://github.com/keymanapp/keyman/assets/19683143/30e2aabb-1fb0-49aa-85de-148fc75e947a">

### Expected behavior

'Unknown Publisher' message should not appear on the screen during Keyman Developer installation in Windows OS. 

### Keyman apps

- [ ] Keyman for Android
- [ ] Keyman for iPhone and iPad
- [ ] Keyman for Linux
- [ ] Keyman for macOS
- [ ] Keyman for Windows
- [X] Keyman Developer
- [ ] KeymanWeb
- [ ] Other - give details at bottom of form

### Keyman version

17.0.186-alpha

### Operating system

Windows 10 OS, Windows 11 OS (VM)

**Comment by mcdurdin** [comment]
Build log is: https://build.palaso.org/buildConfiguration/Keyman_Developer_Release/411853?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true

This appears to be a bug in the signing script:
```
[02:08:18]W:	 [Step 1/8] /usr/local/bin/sign-digest: 25: Syntax error: "(" unexpected
```

Longer log snippet:
```log
[02:08:12] :	 [Step 1/8] 	copy /b C:\BuildAgent\work\deb663ed6681f452\keyman\developer\bin\setup.exe + setup.zip keymandeveloper-17.0.186.exe
[02:08:13] :	 [Step 1/8] C:\BuildAgent\work\deb663ed6681f452\keyman\developer\bin\setup.exe
[02:08:13] :	 [Step 1/8] setup.zip
[02:08:13] :	 [Step 1/8]         1 file(s) copied.
[02:08:13] :	 [Step 1/8] The following certificate was selected:
[02:08:13] :	 [Step 1/8]     Issued to: SUMMER INSTITUTE OF LINGUISTICS, INC.
[02:08:13] :	 [Step 1/8] 
[02:08:13] :	 [Step 1/8]     Issued by: Sectigo Public Code Signing CA EV R36
[02:08:13] :	 [Step 1/8] 
[02:08:13] :	 [Step 1/8]     Expires:   Thu Sep 17 19:59:59 2026
[02:08:13] :	 [Step 1/8] 
[02:08:13] :	 [Step 1/8]     SHA1 hash: E9E33765A2C871D4FE730592CCA624D07DFC11CC
[02:08:13] :	 [Step 1/8] 
[02:08:13] :	 [Step 1/8] 
[02:08:13] :	 [Step 1/8] Done Adding Additional Store
[02:08:16] :	 [Step 1/8] Successfully signed: keymandeveloper-17.0.186.exe
[02:08:16] :	 [Step 1/8] 
[02:08:16] :	 [Step 1/8] 
[02:08:16] :	 [Step 1/8] Number of files successfully Signed: 1
[02:08:16] :	 [Step 1/8] 
[02:08:16] :	 [Step 1/8] Number of warnings: 0
[02:08:16] :	 [Step 1/8] 
[02:08:16] :	 [Step 1/8] Number of errors: 0
[02:08:16] :	 [Step 1/8] 
[02:08:18]W:	 [Step 1/8] /usr/local/bin/sign-digest: 25: Syntax error: "(" unexpected
[02:08:19] :	 [Step 1/8] Successfully signed: keymandeveloper-17.0.186.exe
[02:08:19] :	 [Step 1/8] 
[02:08:19] :	 [Step 1/8] 
[02:08:19] :	 [Step 1/8] Number of files successfully Signed: 1
[02:08:19] :	 [Step 1/8] 
[02:08:19] :	 [Step 1/8] Number of warnings: 0
[02:08:19] :	 [Step 1/8] 
[02:08:19] :	 [Step 1/8] Number of errors: 0
[02:08:19] :	 [Step 1/8] 
[02:08:19] :	 [Step 1/8] Trying to timestamp keymandeveloper-17.0.186.exe with http://timestamp.sectigo.com
[02:08:19] :	 [Step 1/8] Successfully timestamped: keymandeveloper-17.0.186.exe
[02:08:19] :	 [Step 1/8] 
[02:08:19] :	 [Step 1/8] 
[02:08:19] :	 [Step 1/8] Number of files successfully timestamped: 1
[02:08:19] :	 [Step 1/8] 
[02:08:19] :	 [Step 1/8] Number of errors: 0
[02:08:19] :	 [Step 1/8] 
[02:08:19] :	 [Step 1/8] remotesign exit code is 0. There were 0 timestamping errors.
```

![image](https://github.com/keymanapp/keyman/assets/4498365/4e50d9b1-90f7-4304-93c8-942555adf60b)

**Comment by mcdurdin** [comment]
Critical: 17.0.186-alpha is completely unsigned. Considering removing this build from the download servers.

**Comment by mcdurdin** [comment]
I have verified that signing of 17.0.185 appears to be okay.

**Comment by mcdurdin** [comment]
17.0.186 has been removed from download server.

**Comment by mcdurdin** [comment]
Note, the broken unsigned versions are still available for a while on build infrastructure at:
* https://build.palaso.org/buildConfiguration/Keyman_Developer_Release/411853?buildTab=artifacts#%2Frelease 
* https://build.palaso.org/buildConfiguration/Keyman_Build/411852?buildTab=artifacts#%2Frelease

**Comment by mcdurdin** [comment]
@tim-eves has fixed the code signing script on the signing server.

I am continuing to iterate on the fix in #9695.
